rasa/train.py

```diff
@@ -39,7 +41,30 @@ def train(
     core_additional_arguments: Optional[Dict] = None,
     nlu_additional_arguments: Optional[Dict] = None,
     loop: Optional[asyncio.AbstractEventLoop] = None,
+    training_in_chunks: bool = False,
+    number_of_chunks: int = 5,
 ) -> Optional[Text]:
+    """Trains a Rasa model (Core and NLU).
```

Comment:
Haha, seems we both added docstrings in separate PRs 🙈 https://github.com/RasaHQ/rasa/pull/7358/commits/0d4527e55233033c53c093a6c5a1450a9d9b1990

Reply:
Let's see who will merge first 😄